testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -566,3 +566,18 @@ def test_alignto_reorder_atomgroups():
     ref = u.atoms[[3, 2, 1, 0]]
     rmsd = align.alignto(mobile, ref, select='bynum 1-4')
     assert_allclose(rmsd, (0.0, 0.0))
+
+def test_alignto_same_universe():
```

Comment:
I might suggest adding "deprecation_warning" as part of this name.

Purely on the basis that we often forget to deal with deprecations and having the words "deprecate" or "deprecation" makes it really easy for us to grep for and go "o yeah we need to deal with this".